app/gateway/database/postgres/accounts/account_test.go

```diff
@@ -0,0 +1,135 @@
+package postgres_account
+
+import (
+	"context"
+	"stoneBanking/app/domain/entities/account"
+	"stoneBanking/app/domain/types"
+	"testing"
+	"time"
+)
+
+func Test_Create(t *testing.T) {
+	t.Run("Conta criada com sucesso - Dados corretos", func(t *testing.T) {
+		account := account.Account{
+			ID:         "d3280f8c-570a-450d-89f7-3509bc84980d",
```

Comment:
Seria bom que o `ID` fosse um `int` e tu tivesse como `uuid` uma chave externa, assim tu tem as vantagens dos dois modelos ao máximo e um pouco mais. Tem uma explicação boa nesse [artigo](https://tomharrisonjr.com/uuid-or-guid-as-primary-keys-be-careful-7b2aa3dcb439).

Reply:
Acho interessante usar ids gerados pelo próprio banco, por questão de simplicidade e performance.

Reply:
Achei interessante a ideia de separar o ID (Númerico) do UUID, vou fazer uma Issue separada para isso.
Obrigado pela sugestão.